root/components/forms.tt

```diff
@@ -358,3 +358,20 @@
 [%- MACRO external_links_editor BLOCK -%]
   <div id="external-links-editor-container"></div>
 [%- END -%]
+
+[%- MACRO removal_help(extra_help) BLOCK -%]
+  <div id="removal-help">
+    <p>
+      [% l('Are you sure you wish to remove {entity} from MusicBrainz?', { entity => link_entity(entity) }) %]
+    </p>
+    <p>
+      [% l('If it is a duplicate, {doc_merge|you should probably merge it instead}.', { doc_merge => doc_link('Merge_Rather_Than_Delete') }) %]
+      [% l('If it just has some small errors, it’s usually better to just fix those.') %]
```

Comment:
Should this just be merged with the previous string? (Or if they're meant to be on separate lines, maybe in a separate `<p>` tag.)

Reply:
(Unless we're using/plan to use the strings separately. Otherwise maybe combining them will be better for context.)

Reply:
I separated them because I thought they looked more clear like that, but you might be right :)